bundle/compliance/cis_gcp/rules/cis_4_1/test.rego

```diff
@@ -0,0 +1,37 @@
+package compliance.cis_gcp.rules.cis_4_1
+
+import data.cis_gcp.test_data
+import data.compliance.policy.gcp.data_adapter
+import data.lib.test
+
+test_violation {
+	eval_fail with input as rule_input({"name": "test-instance", "serviceAccounts": [{"email": "12345678-compute@developer.gserviceaccount.com"}]})
+	eval_fail with input as rule_input({"name": "test-instance", "serviceAccounts": [{"email": "legit-sa@developer.gserviceaccount.com"}, {"email": "12345678-compute@developer.gserviceaccount.com"}]})
```

Comment:
do we need both?

Reply:
IMO, yes.
The second test verifies that we check all the attached SA and in case one is not permitted - we fail.